dapps/frenemies/package.json

```diff
@@ -18,6 +18,7 @@
     "@mysten/wallet-kit": "workspace:*",
     "@tanstack/react-query": "^4.22.0",
     "@tanstack/react-query-devtools": "^4.22.0",
+    "bignumber.js": "^9.1.1",
```

Comment:
Are you sure we need this buddy? Won't wrapping into BigInt solve the issue? Besides we use bigint everywhere in our type definitions.